os/drivers/audio/alc5658.c

```diff
@@ -517,7 +525,7 @@ static void alc5658_set_i2s_samplerate(FAR struct alc5658_dev_s *priv)
  ****************************************************************************/
 static int alc5658_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR struct audio_caps_s *caps)
 {
-#if !defined(CONFIG_AUDIO_EXCLUDE_VOLUME) || !defined(CONFIG_AUDIO_EXCLUDE_TONE)
+#if !defined(CONFIG_AUDIO_EXCLUDE_SPEAKER_VOLUME) || !defined(CONFIG_AUDIO_EXCLUDE_MIC_GAIN)
```

Comment:
EXECLUDE_TONE needed to handle AUDIO_FU_TREBLE...and so on.

Reply:
Right. But I removed it since EXCLUDE_TONE is not used within this function currently.